lib/rubyGemsApi.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/gemInfo.rb'
+require 'faraday'
+require 'faraday/net_http'
+require 'json'
+Faraday.default_adapter = :net_http
+
+class RubyGemsApi
+  def initialize
+    @base_uri = "https://rubygems.org/api"
+  end
+
+  def get_gem_by_name(name)
```

Comment:
I'm thinking here, why did you prefix the method with `get_`? I'm hoping that you are aware that this is ruby and not java 😄

Reply:
What would a more suitable name be?

Reply:
An answer was given by the question :D